internal/hardware/hardware.go

```diff
@@ -89,3 +89,27 @@ func MapHardware(hardware *models.HardwareInfo) *v1alpha1.Hardware {
 	}
 	return &hw
 }
+
+func MapLabels(hardware *models.HardwareInfo) map[string]string {
+	if hardware == nil {
+		return nil
+	}
+	var labels = make(map[string]string)
+
+	labels["feature.device/hostname"] = hardware.Hostname
+
+	cpu := hardware.CPU
+	if cpu != nil {
+		labels[fmt.Sprintf("feature.device/cpu.%s", cpu.Architecture)] = "true"
+		labels[fmt.Sprintf("feature.device/cpu.%s", cpu.ModelName)] = "true"
```

Comment:
can we assure the value of `cpu.ModelName` is in lowercase?

Reply:
@pkliczewski
according to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ it can be upper case. If use the value and not the name then it can be anything and we can use the original string